File: Python-Basics/salary.py
open_tabs = int(input())
salary = int(input())

fine = 0
for tabs in range(open_tabs):
    site_names = input()
    if site_names == "Facebook":
        fine += 150
    elif site_names == "Instagram":
        fine += 100
    elif site_names == "Reddit":
        fine += 50
    else:
        fine += 0
diff = abs(salary - fine)
if fine > salary or salary == 0:
    print("You have lost your salary")
else:
    print(f"{round(diff)}")

File: JS-Basics/For Loop Exercise/salary.js
function salary(input){
    let count = Number(input[0]);
    let salary = Number(input[1]);
    let inputL = input.length
    for(let i=2; i<inputL; ++i){
        let currentTab = input[i]

        if(currentTab === "Facebook"){
            salary -= 150;
        }
        else if(currentTab === "Instagram"){
            salary -= 100;

        }
        else if(currentTab === "Reddit"){
            salary -= 50;
            
        }
        if(salary <=0){
            console.log("You have lost your salary.")
            break
        }
    }
    if(salary >0){
        console.log(salary)
    }
}
salary(["3",
"500",
"Facebook",
"Stackoverflow.com",
"softuni.bg"])

;
